### PR TITLE
add reference to tool command units to close issue #252

### LINKED ISSUE #252: create a reference for tool command 'units'
# Advanced Homework 7 Submission

Name: Anna Gray
uniquename: annagray

### PR DESCRIPTION
**Pull request by annagray**
# Advanced HW 7 submission

Closes #252

Name: Anna Gray
Uniqname: annagray

### CODE REVIEW

File: _commands/tools/units.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+~~~ bash
+$ units [-f unitsfile] [-q] [-v] [from-unit to-unit]...
+~~~
+
+<!--more-->
+
+The units are defined in an external data file. You can use the extensive data file that comes with this program, or you can provide your own data file to suit your needs. You can use the program interactively with prompts, or you can use it from the command line.
+
+will list the value you will need to multiply (*) or divide (/) your original number by to convert to your desired units listed.
```
Comment (mterwill) [inline _commands/tools/units.md]: This line seems kind of orphaned? Is it supposed to be below 19-21?

Comment (annagray) [inline _commands/tools/units.md]: Updated - thanks for the catch, I didn't realized I had changed the ordering without making sure the full sentences still made sense.

File: _commands/tools/units.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+<!--more-->
+
+The command above will list the value you will need to multiply (*) or divide (/) your original number by to convert to your desired units listed.
+
+The units are defined in an external data file. You can use the extensive data file that comes with this program, or you can provide your own data file to suit your needs. You can use the program interactively with prompts, or you can use it from the command line.
+
+~~~ bash
+$ units
+~~~The command above will list the value you will need to multiply (*) or divide (/) your original number by to convert to your desired units listed.
```
Comment (mterwill) [inline _commands/tools/units.md]: This needs to be moved to a newline; it's not rendering correctly. You also have the same line twice. It is also on `15`.

image

File: _commands/tools/units.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+$ units --file filename
+~~~
+
+Use filename as the units data file rather than the default units data file `units.dat'.
+
+~~~bash
+$ units --check-verbose
+~~~
+
+Like the `-check` option, this option prints a list of units that cannot be reduced.  But to help find unit  definitions that cause endless loops, it lists the units as they are checked.  If `units' hangs, then the last unit to be printed has a bad definition.
```
Comment (mterwill) [inline _commands/tools/units.md]: Please make the backticks here consistent:

~~~
\`units'

File: _commands/tools/units.md [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+~~~
+
+will display version information and exit.
+
+~~~ bash
+$ units -f filename
+$ units --file filename
+~~~
+
+Use filename as the units data file rather than the default units data file `units.dat'.
```
Comment (mterwill) [inline _commands/tools/units.md]: Please make the backticks here consistent:

~~~
\`units.dat'

**Comment by mterwill** [comment]
Looking good! Merging + giving you credit.

**Pull request merged**